### PR TITLE
Finalize-header-feature

### PR DESCRIPTION
**Pull request by LadyAudrey**
added Link components for buttons and finalized css spacing

### CODE REVIEW

**Review by kensonjohnson — CHANGES_REQUESTED** [review]
I cannot test the header in it's current state. My suggestion is to remove the extra content from the page and start building the page back component by component, from top to bottom.

File: app/components/Header/MobileHeader.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+          src={"/misc/chillNoBG.png"}
+          height={50}
+          width={50}
+          alt="hamburger"
+          className="h-auto"
+        />
+        <h1>Thomas Burger</h1>
+      </div>
+      <div onClick={toggleNav}>
+        {navOpen ? (
```
Comment (kensonjohnson) [inline app/components/Header/MobileHeader.tsx:27]: Let's decouple the nav from the state of the menu icon.

File: app/components/Header/MobileHeader.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+              fill="none"
+              stroke="currentColor"
+              strokeWidth="2"
+              strokeLinecap="round"
+              strokeLinejoin="round"
+            >
+              <line x1="18" y1="6" x2="6" y2="18" />
+              <line x1="6" y1="6" x2="18" y2="18" />
+            </svg>
+            <div className="absolute flex flex-col top-20 right-5">
```
Comment (kensonjohnson) [inline app/components/Header/MobileHeader.tsx]: This would do well as a `<nav>` instead.

**Review by kensonjohnson — CHANGES_REQUESTED** [review]

File: app/components/Header/MobileHeader.tsx (hunk 1 above)
Comment (LadyAudrey) [reply app/components/Header/MobileHeader.tsx:27]: That makes sense. What would you like me to attach it to?

**Review by kensonjohnson — APPROVED** [review]

**Pull request merged**